client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/provisioning/test/index.tsx

```diff
@@ -41,7 +41,7 @@ describe( 'Provisioning', () => {
 		const message = container.querySelector( '.migration-instructions-provisioning__message' );
 
 		expect( message?.textContent ).toBe(
-			'Sorry, we couldn’t finish setting up your site. Please, contact support.'
+			'Sorry, we couldn’t finish setting up your site. Please contact support.'
```

Comment:
Fix suggested in https://github.com/Automattic/wp-calypso/pull/92557#issuecomment-2228913954